src/octopeer-github/content/Types/ElementID.ts

```diff
@@ -1,10 +1,68 @@
 /**
  * Created by Youri on 08/05/2016.
- */
-/**
  * Class to hold elementID's so they can be a semantically different data type.
  */
 class ElementID {
+
+    /** Instantiated ElementID for the MergePullRequestButton. */
+    public static get MERGEPRREQUEST() { return new ElementID(101); }
+    /** Instantiated ElementID for the ClosePullRequestButton. */
+    public static get CLOSEPRREQUEST() { return new ElementID(102); }
+    /** Instantiated ElementID for the CancelInlineCommentButton. */
+    public static get CANCELINLINECOMMENT() { return new ElementID(103); }
+    /** Instantiated ElementID for the ConfirmInlineCommentButton. */
+    public static get CONFIRMINLINECOMMENT() { return new ElementID(104); }
+    /** Instantiated ElementID for the CreateInlineCommentButton. */
+    public static get CREATEINLINECOMMENT() { return new ElementID(105); }
+    /** Instantiated ElementID for the EditPullRequestNameButton. */
+    public static get EDITPRNAME() { return new ElementID(106); }
+    /** Instantiated ElementID for the SavePullRequestNameButton. */
+    public static get SAVEPRNAME() { return new ElementID(107); }
+    /** Instantiated ElementID for the CancelEditPullRequestNameButton. */
+    public static get CANCELEDITPRNAME() { return new ElementID(108); }
+    /** Instantiated ElementID for the EditCommentButton. */
+    public static get EDITCOMMENT() { return new ElementID(109); }
+    /** Instantiated ElementID for the AddEmoticonButton. */
+    public static get ADDEMOTICON() { return new ElementID(110); }
+    /** Instantiated ElementID for the ShowCIChecksToggleButton. */
+    public static get SHOWCICHECKSTOGGLE() { return new ElementID(111); }
+    /** Instantiated ElementID for the ShowCIDetailsButton. */
+    public static get SHOWCIDETAILS() { return new ElementID(112); }
+    /** Instantiated ElementID for the CommentPullRequestButton. */
+    public static get COMMENTPR() { return new ElementID(113); }
+    /** Instantiated ElementID for the ConversationTab. */
+    public static get CONVERSATIONTAB() { return new ElementID(201); }
+    /** Instantiated ElementID for the CommitsTab. */
+    public static get COMMITSTAB() { return new ElementID(202); }
```

Comment:
Yeah, let's `STAB` some `COMMIT`s :D 
Suggestion: Use underscores to indicate spaces in the names. So things like:
- `COMMITS_TAB` (or `TAB_COMMITS`)
- `SHOW_CI_DETAILS`
- `PR_CREATOR`
- `MERGE_PR`
- `SAVE_PR_NAME`